src/main/java/bio/terra/cloudfunctions/common/ContentHandler.java

```diff
@@ -0,0 +1,33 @@
+package bio.terra.cloudfunctions.common;
```

Comment:
I'm not sure that we need this class yet, but I could be convinced that we do.  I feel like translate() should have a return value if we include this class

Reply:
Are you referring to a return value of generic type specified by the application?